utilities/convert_frosst/smirffishFrcmod.parm99Frosst.txt

```diff
@@ -131,22 +132,23 @@ BOND
 ANGL
 # begin section from parm99
 [*:1]~[*:2]~[*:3]         80.0      120.0   smirff generic is sp2 hybridized
-# Csp3 general
-[*:1]~[#6X4:2]-[*:3]      50.  109.5   parm99 generic Csp3
-[#1:1]-[#6X4:2]-[#1:3]    35.  109.5   parm99 generic Csp3
-# Csp3 in 3-membered ring
+# Any atom in 3-membered ring
 [*;r3:1]1~;@[*;r3:2]~;@[*;r3:3]1 350.  60.     Frosst generic any atom within the 3-membered ring
 [*;r3:1]~;@[*;r3:2]~;!@[*:3] 100.  118.    Frosst generic 3ringAtom-3ringAtom-substit
 [*:1]~;!@[*;r3:2]~;!@[*:3]  100.  113.    Frosst generic substit-3ringAtom-nonHsubstit
 [#1:1]-[*;r3:2]~;!@[*:3]    50.  113.    Frosst generic substit-3ringAtom-Hsubstit
-# generic Csp2 120.
-[*:1]~[#6X3:2]~[*:3]      70.  120.    parm99 generic Csp2
-[#1:1]-[#6X3:2]~[*:3]     50.  120.    parm99 generic Csp2
-[#1:1]-[#6X3:2]-[#1:3]    35.  120.    parm99 generic Csp2
 # fused het6-het5 bridgehead outer angle
 [*;r6:1]~;@[*;r5:2]~;@[*;r5;R2:3] 70.0  130.    parm99 generic fused het6-het5 bridgehead outer angle
 # het5 outer angle
 [*:1]~;!@[*;r5:2]~;@[*;r5:3]  70.0  125.    Frosst generic het5 substituent outer angle
+# Begin carbons - starting from most generic
```

Comment:
This section was in the middle of the generic ring section before. The numbers haven't changed. Noted for the recorded nothing wrong here.